openemory/harvest/entrez.py

```diff
@@ -95,7 +96,8 @@ def _query(self, base_url, qargs, response_xmlclass):
         # policies.
         qurl = base_url + urlencode(qargs)
         logger.debug('EntrezClient querying: ' + qurl)
-        return xmlmap.load_xmlobject_from_file(qurl,
+        target_file = urllib2.urlopen(qurl)
```

Comment:
Think this looks good.  I'm assuming that at this point in the code it will always open a URL and never a local file?